prov/gni/test/freelist.c

```diff
@@ -147,7 +147,7 @@ Test(gnix_freelist, freelist_zero_refill_test)
 	cr_assert(_gnix_fl_empty(&fl), "Freelist not empty");
 
 	ret = _gnix_fl_alloc(&elems[num_elems], &fl);
-	cr_assert_eq(ret, -FI_ECANCELED, "Freelist empty and refill size is zero");
+	cr_assert_eq(ret, -FI_ECANCELED, "Freelist empty but refill size is zero");
```

Comment:
I don't think this fixes the description issue. If this assertion is raised, it is because we saw a different error code (FI_SUCCESS, or some other -FI_* error code). 

Describe the assertion by saying what was supposed to happen and did not. What was expected?